public/gallery/photos/create_thumbs.sh

```diff
@@ -1,16 +1,36 @@
 #!/bin/bash
-THUMBS_FOLDER=2017/thumb/*
-for file in 2017/full/*
+set -eo pipefail
+
+#
+# This script generates thumbnails for a year's gallery photos.
+# Run it like this:
+#
+#     cd /path/to/brickhack.io/public/gallery/photos/
+#     bash create_thumbs.sh
+#
+#For Windows users, open docker and run:
```

Comment:
Nit: Docker is our de-facto environment, so these could just replace line 8 & 9 entirely